README.md

```diff
@@ -35,12 +35,6 @@ The endpoints and known consumers of this application's APIs are documented in [
 - **Sub-topic**: a group of content within a topic. (A deprecated name for this is
 "specialist sub-sector".)
 
-### Browse pages
-
-- **Root browse page**: [gov.uk/browse](https://www.gov.uk/browse)
-- **Top level browse page**: [gov.uk/browse/benefits](https://www.gov.uk/browse/benefits)
-- **Second level browse page**: [gov.uk/browse/benefits/entitlement](https://www.gov.uk/browse/benefits/entitlement)
```

Comment:
Hmmm I did have some misgivings about removing this, as it could be quite useful to have an awareness of the different types of browse pages. But agreed it can be discovered at point of need. And the [vocabulary is already in the code](https://github.com/alphagov/collections/blob/c63209674bae27007a08ea172e7befb56fe500b5/config/routes.rb#L22-L24), so we're not particularly losing anything here.